cli/stork-publisher-agent.go

```diff
@@ -31,8 +31,8 @@ const ClockUpdatePeriodFlag = "clock-update-period"
 const DeltaUpdatePeriodFlag = "delta-update-period"
 const ChangeThresholdPercentFlag = "change-threshold-percent"
 const IncomingWsPortFlag = "incoming-ws-port"
-const OutgoingWsUrlFlag = "outgoing-ws-url"
-const OutgoingWsAuthFlag = "outgoing-ws-auth"
+const StorkRegistryBaseUrlFlag = "stork-registry-base-url"
```

Comment:
defaults to our prod API - only making it configurable so we can hit the dev registry